whois-query/src/main/java/net/ripe/db/whois/query/pipeline/WhoisEncoder.java

```diff
@@ -22,13 +24,16 @@ public class WhoisEncoder extends MessageToMessageEncoder<Object> {
     private static final int DEFAULT_BUFFER_SIZE = 1024;
     private static final byte[] OBJECT_TERMINATOR = {'\n'};
 
+    public static final AttributeKey<String> CHARSET_ATTRIBUTE = AttributeKey.valueOf("charset");
```

Comment:
Isn't `"charset"` already defined in `QueryFlag` ?

Reply:
I will use QueryFlag name